src/commands/scanner/run.ts

```diff
@@ -192,6 +181,14 @@ export default class Run extends ScannerCommand {
 			throw new SfdxError(messages.getMessage('validations.mustTargetSomething'), null, null, this.getInternalErrorCode());
 		}
 
+		if (this.flags.tsconfig && this.flags.eslintconfig) {
```

Comment:
New validation to throw a helpful error message when `--tsconfig` and `--eslintconfig` is provided in the same command.